CMakeLists.txt

```diff
@@ -21,16 +21,10 @@
 PROJECT(mudlet)
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
Will accept this at present and I'll see if http://stackoverflow.com/a/31010221/4805858 permits a work around in the future. (I think I mentioned this in the original commit that introduce the CMake 3.1 requirement!)